nullaway/src/main/java/com/uber/nullaway/ErrorBuilder.java

```diff
@@ -302,12 +307,39 @@ boolean symbolHasSuppressWarningsAnnotation(Symbol symbol, String suppression) {
     return false;
   }
 
-  static String errMsgForInitializer(Set<Element> uninitFields) {
+  static int GetLineforField(Element uninitField, VisitorState state) {
+    Tree fieldTree = getTreesInstance(state).getTree(uninitField);
+    DiagnosticPosition position = (DiagnosticPosition) fieldTree;
+    TreePath path = state.getPath();
+    JCCompilationUnit compilation = (JCCompilationUnit) path.getCompilationUnit();
+    JavaFileObject file = compilation.getSourceFile();
+    DiagnosticSource source = new DiagnosticSource(file, null);
+    return source.getLineNumber(position.getStartPosition());
+  }
+
+  static String errMsgForInitializer(Set<Element> uninitFields, VisitorState state) {
```

Comment:
Technically not needed for non-public methods, but a Javadoc comment makes sense to me here (not needed for `GetLineforField`, since the name is fairly self-documenting, although see comment above).

Reply:
Yes, makes sense.  I think this method existed before this PR, but good to add some docs now.

Reply:
Sure, that was our issue, then, not a new one, but it's still worth fixing/improving :)

Reply:
Sounds great! I will add a javadoc comment for errMsgForInitializer!